data/Nigeria/Senate/sources/instructions.json

```diff
@@ -21,6 +21,21 @@
         "from": "group-wikidata",
         "source": "manual/group_wikidata.csv"
       }
-    }
+    },
+    {
+      "file": "morph/shineyoureye.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "struan/nigeria-shineyoureye",
+        "query": "SELECT *, NULL as position FROM data WHERE position = 'Senator'"
```

Comment:
I think this will need to alias the `identifier__pombola_slug` column on the way through. At the minute that creates data in the format:

```json
      "identifiers": [
         {
           "identifier": "samuel-egwu",
           "scheme": "pombola_slug"
         }
       ],
```

But 'pombola_slug' doesn't really describe what the source is. I suspect this should just be `source__shineyoureye`.